package.json

```diff
@@ -27,16 +31,18 @@
     "prepare": "rm -rf dist && mkdir dist && tsc"
   },
   "dependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
     "@emotion/react": "^11.11.1"
   },
   "devDependencies": {
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "typescript": "^4.9.5"
   },
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1"
```

Comment:
해당 패키지를 peerDependencies로 이동시킨 이유가 궁금해요!

Reply:
저번 step1 때 dist를 붙여야만 작동한다고 한 거 기억나시나요..?! 저도 갑자기 그런 현상이 생겨서 좀 찾아봤었는데요
<img width="704" alt="image" src="https://github.com/woowacourse/layout-component/assets/57815133/896bdf2b-9dde-49a6-9a18-9ac866ae3a7a">

이런 오류가 떴었는데 3번 You might have more than one copy of React in the same app. 보니까 dependency로 react를 넣어놔서 react가 또 설치되서 두개가 되어있더라구요.

[공식문서](https://react-ko.dev/warnings/invalid-hook-call-warning#duplicate-react) 보면 원인이 사용중인 라이브러리가 peerDependency가 아닌 dependency로 의존성을 잘못 지정하고 있을 수 있다. 

> For example, maybe a library you’re using incorrectly specifies react as a dependency (rather than a peer dependency).

라고 나와있는데 이걸 해결하기 위해서  peerDependencies에 React를 명시해서 이 패키지는 React에 종속되지만 자동으로 설치되지는 않으며 프로젝트 내에서 명시적으로 관리해아한다는 것을 나타내기 위해 이동하였습니다!